### PR TITLE
Grails 6.2.2 release blog post

### PR DESCRIPTION
**Pull request by jamesfredley**

### CODE REVIEW

File: posts/2024-11-20-introducing-grails-6-2-2.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+## Upgrading Your Existing Applications to Grails 6.2.2
+
+If you already have a Grails application and want to upgrade to the latest version, follow these steps:
+
+1. Open the project in your favorite IDE (preferably JetBrains' IntelliJ IDEA).
+2. Update your application's `gradle.propreties` file to specify Grails 6.2.2 as the desired version.
+
+    ````properties
+    grailsVersion=6.2.2
+    grailsGradlePluginVersion=6.2.2
```
Comment (matrei) [inline posts/2024-11-20-introducing-grails-6-2-2.md:95]: Should this be `6.2.3`?

File: posts/2024-11-20-introducing-grails-6-2-2.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+image: grails-blog-index-3.png
+---
+
+# [%title]
+
+[%author]
+
+[%date]
+
+The Grails Foundation is excited to announce the release of Grails framework 6.2.2. This latest version delivers critical bug fixes and reintroduces key features based on community feedback.
```
Comment (matrei) [inline posts/2024-11-20-introducing-grails-6-2-2.md:15]: Are there any `critical` bug fixes in `6.2.2`?

**Review by matrei — COMMENTED** [review]
Great! I have made 2 comments.

File: posts/2024-11-20-introducing-grails-6-2-2.md (hunk 2 above)
Comment (jamesfredley) [reply posts/2024-11-20-introducing-grails-6-2-2.md:15]: I'll remove that word.

File: posts/2024-11-20-introducing-grails-6-2-2.md (hunk 1 above)
Comment (jamesfredley) [reply posts/2024-11-20-introducing-grails-6-2-2.md:95]: I'll update.

**Review by matrei — APPROVED** [review]

**Review by codeconsole — APPROVED** [review]

**Pull request merged**